app/models/ems_cluster.rb

```diff
@@ -232,11 +232,11 @@ def all_resource_pools_with_default
 
   # Parent relationship methods
   def parent_folder
```

Comment:
@Fryguy EmsCluster ResourcePool and Host all have identical parent_folder and parent_datacenter methods, is there a good common place to pull these into?  Seems too specific for the relationship mixin.